linux/xkb/symbols/3l

```diff
@@ -78,3 +78,109 @@ xkb_symbols "cros" {
         symbols[Group1] = [ Tab ]
     };
 };
+
+
+// This is an alternative implementation, inspired by the original.
+// The main differences are:
+// 1. This layout implements just what appears in
+// https://github.com/jackrosenthal/threelayout/blob/master/docs/standard_three.png
+// (the original defines but does not document a bunch of greek letters, etc.)
+// 2. This layout uses CapsLock as the Cur modifier.
+// 3. This layout uses the Shift keys as the Sym modifiers.
+// 4. This layout uses the key just to the left of Enter
+// (the Sym key in the original layout) as "a funny kind of CapsLock":
+// - pressing it once capitalizes the next character
+// - pressing it twice acts as a regular CapsLock
+// - instead of double press, one can hold down either CapsLock (aka Cur)
+// or Shift (aka Sym) while pressing the "funny CapsLock" to get the
+// "regular CapsLock" behaviour
+// 5. In this layout, Escape is on Cur-Space as well as on Sym-Space
+partial alphanumeric_keys modifier_keys
+xkb_symbols "alt" {
+    name[Group1] = "English (3l alternative)";
+
+    key.type[Group1] = "ONE_LEVEL";
```

Comment:
Can you include 3l(basic) instead and drop the repeated base layout? Or is there a specific reason for not including the Greek and extended symbols?

Reply:
Well, this is my first foray into working with xkb, so I wanted something I can easily grok.  The EIGHT_LEVEL_SEMIALPHABETIC key type looked a bit overwhelming on the first go-round.

Reply:
Yeah, xkb is a little bit cryptic. If you can get it working with the include, it'll be easier to send upstream to xkeyboard-config.